boards/common/weact-f4x1cx/Kconfig

```diff
@@ -16,6 +16,7 @@ config BOARD_COMMON_WEACT_F4X1CX
     select HAS_PERIPH_TIMER
     select HAS_PERIPH_UART
     select HAS_PERIPH_USBDEV
+    select HAS_TINYUSB_DEVICE
```

Comment:
Kconfig won't be happy with this. This board and all the other boards with a signle USB interface was part of PR #18998 where I also changed the approach of the STDIO backend selection for Kconfig in commit https://github.com/RIOT-OS/RIOT/pull/18998/commits/b2c855afe282e39f57bc1fbd6f6295d4f8a75e41.

But with the current we woul have to use the STDIO backend selection as for `stm32f4discovery` or `esp32s-wemos-mini` dependent on what the default selection should be.

Reply:
Ah true it's the problem again that would require a `weact-f4x1cx.conf` just to make Kconfig CI happy but without solving the issue on the Kconfig side.